frontend-student/src/components/coursePage/AISummaryComponent.jsx

```
@@ -0,0 +1,238 @@
+      prompt += 'formatted as bullet points, ';
+    }
+
+    if (summaryType.highlights) {
+      prompt += 'focusing on the main highlights, ';
+    }
+
+    if (summaryType.actionItems) {
+      prompt += 'including actionable takeaways, ';
+    }
+
+    return prompt.slice(0, -2) + '.';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)